### PR TITLE
[Device-Registry] Split activity endpoint

### PR DESCRIPTION
**Pull request by ja-odur**
#### Summary of Changes (What does this PR do?)
- Split the deploy, recall and maintain endpoint /api/v1/devices/ts/activity?type=<ACTIVITY TYPE>&tenant=<ORGANISATION> into three different endpoints.
- The motivation is to resolve some of the current deployments issues - [247](https://github.com/airqo-platform/AirQo-frontend/issues/247), [246](https://github.com/airqo-platform/AirQo-frontend/issues/246), and possibly [244](https://github.com/airqo-platform/AirQo-frontend/issues/244)
- The new endpoints and respective sample request data
 -- **Recall device endpoint** `POST http://localhost:3000/api/v1/devices//ts/activity/recall?tenant=airqo&deviceName=6A` **body not required**
 -- **Deploy device endpoint** `POST http://localhost:3000/api/v1/devices//ts/activity/deploy?tenant=airqo&deviceName=6A`
sample body
```
{
      "height": "140",
      "mountType": "pole",
      "powerType": "mains",
      "date": "2020-07-07T16:09:24.611Z",
      "latitude": "0.423",
      "longitude": "0.21",
      "isPrimaryInLocation": false,
      "isUsedForCollocation": false
}
```
 -- **Maintain device endpoint** `POST http://localhost:3000/api/v1/devices//ts/activity/maintain?tenant=airqo&deviceName=6A`
Sample body
```
{
     "date": "2020-07-07T16:09:24.611Z",
    "tags":["blowing", "dusting", "sensor placement"],
     "maintenanceType":"preventive",
     "description": "This is a test log"
}
```

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Fetch this branch locally.
* Install requirements `npm install`
* Run App `npm run stage-mac` (MacOS) or `npm run stage-pc` (Windows Platform)
* Test out the new endpoints stated above.

#### What are the relevant tickets?
- [PLAT-538](https://airqoteam.atlassian.net/browse/PLAT-538)

### CODE REVIEW

**Review by NoahNsimbe — APPROVED** [review]
LGTM

File: src/device-registry/controllers/manage-site.js [hunk 1]
```diff
@@ -22,51 +23,91 @@ const {
...
+  },
+  deploymentFields: [
+      "height",
+      "mountType",
+      "powerType",
+      "date",
+      "latitude",
+      "longitude",
+      "isPrimaryInLocation",
+      "isUserForCollocaton"
```
Comment (sserurich) [inline src/device-registry/controllers/manage-site.js:60]: was this supposed to be isUsedForCollocation. just a check

**Review by Baalmart — CHANGES_REQUESTED** [review]
LGTM, file changes expected and also works as expected.
Just one very small hotfix we need to make as highlighted by @sserurich and I in the comments.,

File: src/device-registry/controllers/manage-site.js (hunk 1 above)
Comment (ja-odur) [reply src/device-registry/controllers/manage-site.js]: @Baalmart @sserurich is has been fixed, thanks.

**Pull request merged**